install_builder/deadline-cloud-for-keyshot.xml

```diff
@@ -101,7 +94,6 @@
             <description>KeyShot Scripts Directory</description>
             <explanation>Path to scripts directory in the KeyShot resources directory. For easiest installation, KeyShot should be installed before installing Deadline Cloud for KeyShot.</explanation>
             <allowEmptyValue>0</allowEmptyValue>
-            <default>Will be detected from the KEYSHOT or KEYSHOT12 environment variables during installation. If neither exists and the KeyShot component is enabled, this must be input.</default>
```

Comment:
Why are we removing this? It is used in CLI help text. If we're worried about it no longer matching the existing functionality, can we update to describe that the env var is only used if the directory exists?

Reply:
The default property is for the default variable value, not really help text though it might have appeared in CLI help in a convenient location. In some cases, this text was being stored as the value for the scripts directory and appeared inside the directory path text box which looked wrong. I'll move the info about the env variables to the CLI help text.

Reply:
With the latest change, the CLI help text looks like this:
![Screenshot 2024-10-10 at 10 36 12 AM](https://github.com/user-attachments/assets/9eff417d-2488-4348-aefc-86b99872ab70)

Reply:
While this is technically intended for default values, it's effectively only used for help text based on the installer definition.

I actually think customers will find it easier to understand with the former version. I find this new version to be confusing because it says that "The default value is detected from the KEYSHOT or KEYSHOT12 environment variables if present." but the default property is missing despite the env vars being present on the machine.

At a minimum, I think we need to clarify that the default value is determined during installation and won't be empty. That's something that isn't clear with an empty default value.

Reply:
This default test now appears in the text box when a scripts directory isn't detected, so we can't reuse it in the same way we were before. I'll clarify the help text about the default being determined at runtime.